service/worker/archiver/activities.go

```diff
@@ -226,6 +243,70 @@ func deleteHistoryActivity(ctx context.Context, request ArchiveRequest) (err err
 	return nil
 }
 
+// deleteBlobActivity deletes uploaded history blobs from blob store.
+// method will retry all retryable operations until context expires.
+// method will always return either: nil, contextTimeoutErr or an error from deleteBlobActivityNonRetryableErrors.
+func deleteBlobActivity(ctx context.Context, request ArchiveRequest) (err error) {
```

Comment:
Can we add a TODO: as part of method comment that says this method should take a list of blobs uploaded once correct heartbeating is supported?

Reply:
Done.